src/App.jsx

```diff
@@ -0,0 +1,49 @@
+import "./App.css";
+import { useState } from "react";
+import { useEffect } from "react";
+import divider from "./assets/divider.svg";
+import dice from "./assets/dice.svg";
+
+function App() {
+  const [adviceNumber, setAdviceNumber] = useState(null);
+  const [adviceText, setAdviceText] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const getAdvice = async () => {
+    try {
+      setIsLoading(true);
+
+      const response = await fetch("https://api.adviceslip.com/advice");
+      const data = await response.json();
+      const slip = data.slip;
+      const number = slip.id;
+      const text = slip.advice;
```

Comment:
`number` and `text` are too generic, consider changing these names, keeping them default is a good idea too, to avoid extra complexity

Example 👇
```suggestion
      const { slip } = data;
      const { id, advice } = slip;
```